packages/ui/client/components/TaskItem.vue

```diff
@@ -23,7 +23,7 @@ const duration = computed(() => {
   >
     <StatusIcon :task="task" mr-2 />
     <div flex items-end gap-2 :text="task?.result?.state === 'fail' ? 'red-500' : ''">
-      <span text-sm truncate font-light>{{ task.name }}</span>
+      <span text-sm truncate font-light>{{ `${(task.type === "suite" && task.projectName) ? `${task.projectName}/` : ''}${task.name}` }}</span>
```

Comment:
I don't like how it's shown. In CLI it's marked as `[${name}]` and has a color

Reply:
@sheremet-va Like this ?
![Screenshot 2023-07-04 204747](https://github.com/vitest-dev/vitest/assets/61521805/82cbd8ee-1be1-470c-8f97-ffa408eac300)

Reply:
yes

Reply:
Yellow color is unreadable in light theme:

<img width="237" alt="Screenshot 2023-07-11 at 09 40 45" src="https://github.com/vitest-dev/vitest/assets/16173870/5dc22432-308a-4d09-b59e-543def6e7b85">

Reply:
@sheremet-va How about we if use a bit of blackish-shade with yellow - `#7c7c05`, green - `#025402` and blue - `rgb(2 2 131)` color ?
![Screenshot 2023-07-24 214457](https://github.com/vitest-dev/vitest/assets/61521805/ce964f42-070c-4652-ab92-c83ea8c4b618)

Reply:
happy-dom still looks unreadable to me

Reply:
@sheremet-va  How's this ?
![Screenshot 2023-07-25 215100](https://github.com/vitest-dev/vitest/assets/61521805/8d5d6241-ad75-4058-8d00-ac593ec02a9a)

Reply:
Looks readable. Is it only for light mode? Won't be readable in dark mode

Reply:
Change colors based on theme, use `isDark` from here https://github.com/vitest-dev/vitest/blob/main/packages/ui/client/composables/dark.ts